Suheyb - Week2

suh3yb opened the pull request.

ufukcalimli commented on Week2/src/readFile.js:
```diff
@@ -0,0 +1,66 @@
+
+const readFile = util.promisify(fs.readFile);
+
+async function fetch() {
```
Good approach but the name of the function is a bit confusing

suh3yb commented:
Hey @ufukcalimli, thank you so much for your time and feedback.
I have changed the confusing function name and deleted unnecessary ternary. Also changed todo_id in ER diagram to todo_list_id.
Best regards
